rd-kt/rd-framework/src/main/kotlin/com/jetbrains/rd/framework/Interfaces.kt

```diff
@@ -38,6 +36,10 @@ interface IProtocol : IRdDynamic {
     val outOfSyncModels: ViewableSet<RdExtBase>
 
     val contexts : ProtocolContexts
+
+    val extCreatedNetworked: RdSignal<Triple<RName, RdId?, Long>>
```

Comment:
`RdSignal` is important here because `localChange` is used. I don't think that cast it every time is better than having a more specific type in `IProtocol`. I can change a type of local signal to `ISignal` though.